lib/awful/widget/watch.lua

```diff
@@ -0,0 +1,84 @@
+---------------------------------------------------------------------------
+--- Watch widget.
```

Comment:
There is complains on Reddit that we have no description for objects, it would be nice to have one (even maybe start to add screenshots with light themes (as the doc is black on white)